updateObsConceptIds.txt

```diff
@@ -0,0 +1,280 @@
+update obs set conceptId=1000041 where conceptId=163181;
+update obs set conceptId=1000042 where conceptId=163182;
+update obs set conceptId=1000043 where conceptId=163183;
+update obs set conceptId=1000044 where conceptId=163184;
+update obs set conceptId=1000045 where conceptId=163185;
+update obs set conceptId=1000046 where conceptId=163186;
+update obs set conceptId=1000047 where conceptId=163187;
+update obs set conceptId=1000048 where conceptId=163188;
+update obs set conceptId=1000049 where conceptId=163189;
+update obs set conceptId=1000050 where conceptId=163190;
+update obs set conceptId=1000051 where conceptId=163191;
+update obs set conceptId=1000052 where conceptId=163192;
+update obs set conceptId=1000053 where conceptId=163193;
+update obs set conceptId=1000054 where conceptId=163194;
+update obs set conceptId=1000055 where conceptId=163195;
+update obs set conceptId=1000056 where conceptId=163196;
+update obs set conceptId=1000057 where conceptId=163197;
+update obs set conceptId=1000058 where conceptId=163198;
+update obs set conceptId=1000059 where conceptId=163199;
+update obs set conceptId=1000060 where conceptId=163200;
+update obs set conceptId=1000061 where conceptId=163201;
+update obs set conceptId=1000062 where conceptId=163202;
+update obs set conceptId=1000063 where conceptId=163203;
+update obs set conceptId=1000064 where conceptId=163204;
+update obs set conceptId=1000065 where conceptId=163205;
+update obs set conceptId=1000066 where conceptId=163206;
+update obs set conceptId=1000067 where conceptId=163207;
+update obs set conceptId=1000068 where conceptId=163208;
+update obs set conceptId=1000069 where conceptId=163209;
+update obs set conceptId=1000070 where conceptId=163210;
+update obs set conceptId=1000071 where conceptId=163211;
+update obs set conceptId=1000072 where conceptId=163212;
+update obs set conceptId=1000073 where conceptId=163213;
+update obs set conceptId=1000074 where conceptId=163214;
+update obs set conceptId=1000075 where conceptId=163215;
+update obs set conceptId=1000076 where conceptId=163216;
+update obs set conceptId=1000077 where conceptId=163217;
+update obs set conceptId=1000078 where conceptId=163218;
+update obs set conceptId=1000079 where conceptId=163219;
+update obs set conceptId=1000080 where conceptId=163220;
+update obs set conceptId=1000081 where conceptId=163221;
+update obs set conceptId=1000082 where conceptId=163222;
+update obs set conceptId=1000083 where conceptId=163223;
+update obs set conceptId=1000084 where conceptId=163224;
+update obs set conceptId=1000085 where conceptId=163225;
+update obs set conceptId=1000086 where conceptId=163226;
+update obs set conceptId=1000087 where conceptId=163227;
+update obs set conceptId=1000088 where conceptId=163228;
+update obs set conceptId=1000089 where conceptId=163229;
+update obs set conceptId=1000090 where conceptId=163230;
+update obs set conceptId=1000000 where conceptId=163231;
```

Comment:
hi @HerbertYiga are you still updating this , because am seeing these new ConceptIds are not incrementing ??

Reply:
163231 and 163140 are both assigned to 1000000